autoPlay.user.js

```diff
@@ -1251,13 +1292,18 @@ function tryUsingAbility(abilityId) {
 	return true;
 }
 
-function tryUsingItem(itemId) {
-	if (!canUseItem(itemId)) {
-		return false;
-	}
-
-	triggerItem(itemId);
-	return true;
+function tryUsingItem(itemId, checkInLane) {
+    if (!canUseItem(itemId)) {
+        return false;
+    }
+    if (typeof checkInLane === "undefined") {
```

Comment:
Yeah, that's why i pass true to tryUsingItem on useWormholeIfRelevant. It's only false if you pass false or don't define it at all.